codemeta.json

```diff
@@ -0,0 +1,44 @@
+{
+    "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+    "@type": "SoftwareSourceCode",
+    "license": "https://spdx.org/licenses/MIT",
+    "codeRepository": "git+https://github.com/cta-observatory/lstmcpipe.git",
+    "dateCreated": "2019-09-09",
+    "datePublished": "2020-05-04",
+    "dateModified": "2021-04-21",
+    "downloadUrl": "https://github.com/cta-observatory/lstmcpipe/archive/refs/tags/v0.4.1.tar.gz",
+    "issueTracker": "https://github.com/cta-observatory/lstmcpipe/issues",
+    "name": "lstmcpipe",
+    "version": "0.4.1",
+    "description": "Scripts to ease the reduction of MC data on the LST cluster at La Palma. With this package, the analysis/creation of R1/DL0/DL1/DL2/IRFs can be orchestrated.",
+    "applicationCategory": "CTA, LST",
+    "releaseNotes": "Working version with\n\n - `lstchain` [v0.6.3](https://github.com/cta-observatory/cta-lstchain/releases/tag/v0.6.3)\n -  `HiPeRTA` [v4.0.0-dev](https://gitlab.in2p3.fr/CTA-LAPP/HiPeRTA/-/tags/v4.0.0-dev)\n     - `HiPeRTA` uses `hdf5-R0` files converted from `simtel` by using `ctapipe_io_mchdf5` [v0.3.1-dev](https://github.com/cta-observatory/ctapipe_io_mchdf5/releases/tag/v0.3.1-dev)",
```

Comment:
Why 0.6.3? We have 0.7.3 in the environment

Reply:
releaseNote should be updated in the moment we'd do another release, and thus update with what everything is working

Reply:
Ah, there hasnt been a release since, ok